packages/components/atoms/f-link/test-utils/component-objects/f-link-selectors.js

```diff
@@ -1 +1 @@
-export const COMPONENT = '[data-test-id="link"]';
+export const COMPONENT = '[data-test-id="link-component"]';
```

Comment:
I think CI build is failing because this data-test-id hasn't been added to the component